### DISCUSSION TITLE
ecGEMs in human1

### DISCUSSION THREAD

**Discussion by PkiwiBird** [SysBioChalmers/Human-GEM#409] [Q&A] [answered]
Is there any reason why the ecGEM models built in human1 article were constrained with total protein instead of individual protein content from Zielenski dataset? Was it because converting from gene symbol to protein id would give many genes to protein matches?

In gecko article the protein/carbohydrates and GAM content are adapted in the yeast model. In human1 it isn't adjusted, is this because of lack of data or any other reason?

**Comment by JonathanRob** [comment, marked as answer]
We chose to constrain total protein rather than individual protein content in the Human1 paper because we were generally trying to reduce the number of constraints/data used for the simulations, since such data is often unavailable.

Similarly, we did not adapt/fit any other parameters (such as GAM) because we were interested in how the most simplified (and therefore accessible) approach would perform, with the assumption that the results could likely be further improved with the incorporation of additional data and parameter values.
